packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -1302,9 +1303,50 @@ - (BOOL)setupWriterForPath:(NSString *)path {
   return YES;
 }
 
+// configure application wide audio session manually to prevent overwriting
+// flag MixWithOthers by capture session, only change category if it is considered
+// as upgrade which means it can only enable ability to play in silent mode or
```

Comment:
> enable ability to play in silent mode

is it desired behavior?

Reply:
System itself is doing this at some point at or after `AVCaptureSession` `addInput` or `addOutput` when `automaticallyConfiguresApplicationAudioSession` is YES which is default, this behaviour was there also before.